bin/straxer

```diff
@@ -155,6 +177,26 @@ def main(args):
     if st.is_stored(args.run_id, args.target):
         logging.info("This data is already available.")
         return 1
+    return st
+
+
+def main(args):
+    logging.basicConfig(
+        level=logging.DEBUG if args.debug else logging.INFO,
+        format='%(asctime)s - %(threadName)s - %(name)s - %(levelname)s - %(message)s')
+
+    logging.info(f"Starting processing of run {args.run_id} until {args.target}")
+
+    # These imports take a bit longer, so it's nicer
+    # to do them after argparsing (so --help is fast)
+    import strax
+    import straxen
+    straxen.print_versions({'strax', 'straxen', args.package})
```

Comment:
```suggestion
    straxen.print_versions(tuple({'strax', 'straxen', args.package}))
```
Otherwise straxer complains

Reply:
Sorry, I just committed but I guess it is fine.

Reply:
I'm absolutely furious that you interfere with my PR!